src/Fixer/Basic/NonPrintableCharacterFixer.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Basic;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * Fixer Zero-width space (ZWSP), Non-breaking space (NBSP) and other invisible unicode symbols.
```

Comment:
```diff
-Fixer
+Removes
```

Also, what about adding links to some doc about these symbols? Would that be useful?